extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -565,6 +590,21 @@
 	'communitypage-deadendpages-hightitle' => 'Please add links to these pages',
 	'communitypage-deadendpages-mediumdescription' => 'Please add links to these pages. Well-linked articles help readers discover more information about your topic and improve search engine rankings.',
 	'communitypage-deadendpages-highdescription' => 'Add links to any important names or words that appear in the first few sentences on these pages.',
+	'communitypage-subheader-welcome' => 'Développons cette communauté ensemble.',
+	'communitypage-admins-welcome-text' => 'Ask us anything. We’d love to hear what brought you to {{SITENAME}}',
+	'communitypage-view-all' => 'Tout afficher',
+	'communitypage-help-policy' => 'Règles de la communauté',
+	'communitypage-todo-module-description' => "Les administrateurs peuvent ici suggérer des pages à améliorer, sur lesquelles les nouveaux contributeurs peuvent commencer. (Administrateurs, n'oubliez pas de fournir des détails sur ce qu'il faut faire.)",
```

Comment:
we don't need this key anymore - it has just been removed in this PR ... (więcej już nie piszę ;P )